platform/core/src/services/DicomMetadataStore/DicomMetadataStore.js

```diff
@@ -211,7 +211,15 @@ const BaseImplementation = {
     let study = _getStudy(StudyInstanceUID);
     if (!study) {
       study = createStudyMetadata(StudyInstanceUID);
+      // Will typically be undefined with a compliant DICOMweb server, reset later
```

Comment:
The study description is not a required field at the series level, so it is possible this will come up blank against a valid DICOMweb service, which means it will fail to apply the tmtv hanging protocol